src/main/groovy/com/marklogic/gradle/task/MlcpTask.groovy

```diff
@@ -16,32 +20,12 @@ import com.marklogic.appdeployer.AppConfig
  */
 class MlcpTask extends JavaExec {
 
-    String host
-    Integer port = 8000
-    String username
-    String password 
-    String database
-    
-    String command
-    String input_file_path
-    String input_file_type
-    String input_file_pattern
-    String input_compressed
-    String document_type
-    String output_collections
-    String delimited_root_name
-    String delimited_uri_id
-    String namespace
-    String options_file
-    String output_file_path
-    String output_uri_prefix
-    String output_uri_replace
-    String output_permissions
-    String transform_module
-    String transform_namespace
-    String transform_param
-    String thread_count
-    
+    @Delegate MlcpBean mlcpBean = new MlcpBean();
+
+    public Logger getLogger() {
+        return Logging.getLogger(MlcpTask.class)
```

Comment:
If I did not override, the method signature of getLogger() from the Delegate MlcpBean conflicted with the signature for getLogger() in MlcpTask.